canonical-web-and-design/ubuntu.com
# Fix blank tutorials page in Firefox

## Done

- Fixed bug in Firefox where it was possible to arrive at a blank tutorials page (see [issue](https://github.com/canonical-web-and-design/ubuntu.com/issues/6842) for how to reproduce)
- Removed duplicated inline JS

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally **in Firefox** at: http://0.0.0.0:8001/tutorials
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Pick any tutorial
- Click your browser's back button
- See that you are taken back to /tutorials

## Issue / Card

Fixes #6842
